server/routes/private.js

```diff
@@ -169,4 +169,31 @@ router.post('/get-workers-for-school', async (req, res) => {
     res.send(workerIds);
 });
 
+// Cancels all the appointments/meetings and updates worker availability to unavailable for a worker for a specific day.
+// Note that appointments/meetings are synonymous, but only appointments will be used in the backend to maintain consistency.
+router.get('/cancel-specific-day', async (req, res) => {
+    const paramSchema = Joi.object({
+        workerId: Joi.number().integer(),
```

Comment:
Hi Amy, is there a reason why the workerId feild validation is does not have a `.required()` addition in the Joi validation?

Reply:
Hi Nancy,
Good catch! The `.required()` clause has been added to the end of this statement.

Reply:
Awesome! Thank you for making the change!